tests/common/widgets/search-box/options.ts

```diff
@@ -0,0 +1,461 @@
+import {
+  createMultiSearchResponse,
+  createSearchClient,
+} from '@instantsearch/mocks';
+import {
+  normalizeSnapshot as commonNormalizeSnapshot,
+  wait,
+} from '@instantsearch/testutils';
+import { screen } from '@testing-library/dom';
+import userEvent from '@testing-library/user-event';
+
+import { skippableDescribe } from '../../common';
+
+import type { SearchBoxWidgetSetup } from '.';
+import type { TestOptions } from '../../common';
+import type { InstantSearch, UiState } from 'instantsearch.js';
+
+function normalizeSnapshot(html: string) {
+  return (
+    commonNormalizeSnapshot(html)
+      // Preact and React transform some SVG attributes differently
+      .replace(/fillRule/g, 'fill-rule')
+      .replace(/strokeWidth/g, 'stroke-width')
+      .replace(/strokeOpacity/g, 'stroke-opacity')
```

Comment:
do both work? I thought that possibly the attributes need to be camelCase

Reply:
JS (Preact) has the first format, React and Vue the second.

Reply:
Yes, but the camelcase name is actually react-specific and only `stroke-opacity` works. https://codepen.io/Haroenv/pen/poGEqpz

We should fix the preact/js output to be kebab-case as it should be, and possibly it should be removed if it doesn't even work?

Reply:
Fixed in 7789ccc59

Reply:
the snapshot "cleaning" also should be removed, right @dhayab?

Reply:
Definitely 😅 on it.